axonius_api_client/api/json_api/enforcements.py

```diff
@@ -300,6 +300,9 @@ def get_model_cls() -> t.Any:
     @marshmallow.pre_load
     def pre_load_fix(self, data, **kwargs):
         """Pass."""
+        # PBUG : Updated by is now coming back as an empty string sometimes.
+        if data.get('updated_by') is "":
+            data['updated_by'] = "{}"
```

Comment:
and we set it to an empty dict string so it can be decoded safely? just want to confirm the downstream behavior

Reply:
Yes.